modules/recommended-node-host-practices.adoc

```diff
@@ -27,6 +27,14 @@ actual container starting. Therefore, a system running 10 pods will actually
 have 20 containers running.
 ====
 
+[NOTE]
+====
+Disk IOPS throttling from the cloud provider might have an impact on the runtime 
+- CRI-O and kubelet. They might get overloaded when there are large number of I/O 
```

Comment:
runtime, including CRI-O and the kubelet. The runtime might

Reply:
Fixed it, thanks.